cypress/integration/getTokenSilently.js

```diff
@@ -20,7 +20,7 @@ describe('getTokenSilently', () => {
     }).as('tokenApiCheck');
 
     whenReady();
-    cy.toggleSwitch('use-form-data');
+
```

Comment:
As we use the node-oidc provider, it only supports form data, which is enabled in `resetTests` now.